src/main/java/ru/spbstu/model/Role.java

```diff
@@ -0,0 +1,35 @@
+package ru.spbstu.model;
+
+import lombok.Data;
+import lombok.experimental.Accessors;
+import org.springframework.security.core.GrantedAuthority;
+
+import javax.persistence.*;
+import java.util.Set;
+
+@Data
+@Accessors(chain = true)
+@Entity
+@Table(name = "roles")
+public class Role implements GrantedAuthority {
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private Long id;
+    private String name;
+    @Transient
+    @ManyToMany(mappedBy = "roles")
+    private Set<User> users;
```

Comment:
It is better to exclude `users` from `equals`, `hashCode` and `toString`

Reply:
Done